packages/ag-charts-community/src/chart/errorBar.ts

```diff
@@ -0,0 +1,125 @@
+import type { Scale } from '../scale/scale';
+import { Group } from '../scene/group';
+import type { Node } from '../scene/node';
+import type { Point } from '../scene/point';
+import { Selection } from '../scene/selection';
+import { Path } from '../scene/shape/path';
+import { OPT_STRING, Validate } from '../util/validation';
+import type { ProcessedData } from './data/dataModel';
+
+export interface ErrorBarPoints {
+    readonly yLowerPoint: Point;
+    readonly yUpperPoint: Point;
+}
+
+export class ErrorBarConfig {
+    @Validate(OPT_STRING)
+    yLowerKey: string = '';
+
+    @Validate(OPT_STRING)
+    yLowerName?: string = undefined;
+
+    @Validate(OPT_STRING)
+    yUpperKey: string = '';
+
+    @Validate(OPT_STRING)
+    yUpperName?: string = undefined;
+
+    constructor(yLowerKey: string, yUpperKey: string) {
+        this.yLowerKey = yLowerKey;
+        this.yUpperKey = yUpperKey;
+    }
+}
+
+export class ErrorBarNode extends Path {
+    public points: ErrorBarPoints = { yLowerPoint: { x: 0, y: 0 }, yUpperPoint: { x: 0, y: 0 } };
+
+    updatePath() {
+        // TODO(olegat) implement, this is just placeholder draw code
+        const { path } = this;
+        this.fill = 'black';
+        this.stroke = 'black';
+        this.strokeWidth = 1;
+        this.fillOpacity = 1;
+        this.strokeOpacity = 1;
+
+        const { yLowerPoint, yUpperPoint } = this.points;
+
+        path.clear();
+        path.moveTo(yLowerPoint.x, yLowerPoint.y);
+        path.lineTo(yUpperPoint.x, yUpperPoint.y);
+        path.closePath();
+    }
+}
+
+export class ErrorBars {
+    public groupNode?: Group = undefined;
+    private selection?: Selection<ErrorBarNode> = undefined;
+    private nodeData: (ErrorBarPoints | undefined)[] = new Array(0);
+
+    createNodeData(
+        parent: Node,
+        processedData: ProcessedData<any>,
+        xIndex?: number,
+        _yIndex?: number, // This is will be used when the scatterplot errorbars are implemented
+        xScale?: Scale<any, any, any>,
+        yScale?: Scale<any, any, any>,
+        config?: ErrorBarConfig
+    ) {
+        this.selection = undefined;
+
+        const { nodeData } = this;
+        const { yLowerKey = undefined, yUpperKey = undefined } = config ?? {};
```

Comment:
```suggestion
        const { yLowerKey, yUpperKey } = config ?? {};
```
nit: it's redundant to assign `undefined` when spreading, that's the default anyhow

Reply:
Done.